WeScan/Common/CIRectangleDetector.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  RectangleDetector.swift
+//  WeScan
+//
+//  Created by Boris Emorine on 2/13/18.
+//  Copyright © 2018 WeTransfer. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+/// Class used to detect rectangles from an image.
+struct CIRectangleDetector {
+  
+    static let rectangleDetector = CIDetector(ofType: CIDetectorTypeRectangle, context: CIContext(options: nil), options: [CIDetectorAccuracy: CIDetectorAccuracyHigh])
+    
+    /// Detects rectangles from the given image on iOS 10.
+    ///
+    /// - Parameters:
+    ///   - image: The image to detect rectangles on.
+    /// - Returns: The biggest detected rectangle on the image.
+    static func rectangle(forImage image: CIImage, completion: @escaping ((Quadrilateral?) -> Void)) {
```

Comment:
Aren't the two static functions on this struct pretty much the same? Can't we only keep the one that returns the result directly, and not through a completion block?

Reply:
This change is by @jcampbell05, i'm not sure exactly why it was done, but I think it's related to making the tests.